pymc/tests/test_distributions.py

```diff
@@ -2291,6 +2291,24 @@ def test_dirichlet_multinomial_vectorized(self, n, a, extra_size):
                 3,
                 np.array([1.29317672, 1.50126157]),
             ),
+            (
+                np.array([5, 4, 3, 2, 1]) / 15,
```

Comment:
These tests don't have a right shape for value. More importantly I don't think the hard coded logp values is very maintainable. Since this PR is all about allowing for batching of alpha I suggest you test it with a np.vectorize function of the base logp, and make it a separate test. You can find some examples in the tests for batched Dirichlet and Multinomial.

Reply:
How can I determine the right shape for value?

Reply:
You can take a random draw from the distribution. Value should have the same shape as the distribution, that is `(*alpha.shape, k + 1,)`

Since you are already checking that the distribution has the right draw shapes, you can simply create a test value for the logp by taking a random draw from the distribution, instead of creating one manually..Thay why is enough to parametrize your tests by providing alphas and size.

Reply:
I now use distribution to generate value.